# SERVICE TYPE: SetGeofence
# ------------------------------------------------------------------------------
# This service creates a geofence (lat, lon, alt) to set in geofencing node

geofencing/Geofence geofence  # geofence to set
---
bool success                # whether the geofence has been set or not
# ------------------------------------------------------------------------------
